locales/en-us.json

```diff
@@ -65,8 +66,16 @@
     "tagline": "What kind of user are you?",
     "changelater": "(Don't worry, you can change this later.)",
     "student_user": "I'm a Student or other Reader",
-    "instructor_user": "I'm an Instructor",
-    "instructor_info": "If you're a Teaching/Research Assistant or otherwise working with an instructor, you can also use the Instructor option."
+    "instructor_user": "I'm an Instructor or other Educator",
+    "instructor_info": "If you're a Teaching/Research Assistant or otherwise working with an instructor, you can also use the Instructor option.",
+    "adapt_special_roles_tagline": "How will you be using ADAPT?",
```

Comment:
Suggest: "Tell us more about your role" or similar. Not all instructor-type users will use ADAPT or even know what it is.